upup/models/cloudup/_aws/topologies/_topology_private/bastion.yaml

```diff
@@ -102,7 +112,7 @@ launchConfiguration/bastion.{{ ClusterName }}:
    - securityGroup/bastion.{{ ClusterName }}
   iamInstanceProfile: iamInstanceProfile/masters.{{ ClusterName }}
   imageId: {{ GetBastionImageId }}
-  instanceType: t2.small
+  instanceType: {{ GetBastionMachineType }}
```

Comment:
@kris-nova Yes it will get a value in any case, as it is handled later  here https://github.com/kubernetes/kops/pull/906/files#diff-0b16cb0df6c9387b2c161f3aeb2e032bR419

And now i have also added an extra check in the function to check if the value comes empty !